From 3e8e2ac66383d592218c719959f90eae26016676 Mon Sep 17 00:00:00 2001
From: Ankit Agarwal <ankit.agarwal@intel.com>
Date: Mon, 10 Jun 2024 14:00:54 +0530
Subject: [PATCH] Remove aidl, libminigbm_gralloc_* and gralloc.* Android.bp

Removed Anrdroid.bp files to make sure aidl, libminigbm_gralloc_* and
gralloc.* are not compiled from AOSP tree.

Tracked-On: OAM-110710
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
---
 Android.bp => Android_bp                               | 0
 cros_gralloc/aidl/{Android.bp => Android_bp}           | 0
 cros_gralloc/gralloc4/{Android.bp => Android_bp}       | 0
 cros_gralloc/mapper_stablec/{Android.bp => Android_bp} | 0
 4 files changed, 0 insertions(+), 0 deletions(-)
 rename Android.bp => Android_bp (100%)
 rename cros_gralloc/aidl/{Android.bp => Android_bp} (100%)
 rename cros_gralloc/gralloc4/{Android.bp => Android_bp} (100%)
 rename cros_gralloc/mapper_stablec/{Android.bp => Android_bp} (100%)

diff --git a/Android.bp b/Android_bp
similarity index 100%
rename from Android.bp
rename to Android_bp
diff --git a/cros_gralloc/aidl/Android.bp b/cros_gralloc/aidl/Android_bp
similarity index 100%
rename from cros_gralloc/aidl/Android.bp
rename to cros_gralloc/aidl/Android_bp
diff --git a/cros_gralloc/gralloc4/Android.bp b/cros_gralloc/gralloc4/Android_bp
similarity index 100%
rename from cros_gralloc/gralloc4/Android.bp
rename to cros_gralloc/gralloc4/Android_bp
diff --git a/cros_gralloc/mapper_stablec/Android.bp b/cros_gralloc/mapper_stablec/Android_bp
similarity index 100%
rename from cros_gralloc/mapper_stablec/Android.bp
rename to cros_gralloc/mapper_stablec/Android_bp
-- 
2.34.1

